test/k8sT/Services.go

```diff
@@ -579,6 +579,59 @@ var _ = Describe("K8sServicesTest", func() {
 				testCurlFromPods(echoPodLabel, url, 5, 0)
 			}
 		})
+
+		curlClusterIPFromExternalHost := func(shouldFail bool) {
+			clusterIP, _, err := kubectl.GetServiceHostPort(helpers.DefaultNamespace, serviceName)
+			Expect(err).Should(BeNil(), "Cannot get service %s", serviceName)
+			Expect(govalidator.IsIP(clusterIP)).Should(BeTrue(), "ClusterIP is not an IP")
```

Comment:
Because these assertions are inside a helper, they should use their offset variant. This way the stacktrace will point back to the `It()` block that called the helper when it fails.

Reply:
Fixed.